hermes/bin/src/cli/playground.rs

```diff
@@ -118,7 +119,12 @@ fn create_one_module_app(
 ) -> anyhow::Result<Application> {
     let vfs_name = [name, "_vfs"].concat();
     let vfs = VfsBootstrapper::new(vfs_dir_path, vfs_name).bootstrap()?;
-    let app = Application::new(ApplicationName::new(name), vfs, vec![module]);
+    let app = Application::new(
+        ApplicationName::new(name),
+        vfs,
+        vec![module],
+        HashMap::new(),
+    );
```

Comment:
I think it's better to create a single-entry `module_registry` for consistency. 
```suggestion
    let module_registry = HashMap::from_iter([(name, module.id())]);
    let app = Application::new(
        ApplicationName::new(name),
        vfs,
        vec![module],
        module_registry,
```
`ApplicationName` is fair to use as module name here, because there is exactly 1 module per app in playground.